### DISCUSSION TITLE
xtraplatform-native repository 404 not found

### DISCUSSION THREAD

**Discussion by alankylee** [ldproxy/ldproxy#725] [Setup] [answered]
Could not GET 'https://dl.interactive-instruments.de/repository/maven-snapshots/de/interactive_instruments/xtraplatform-native/2.1.0-windows-amd64-SNAPSHOT/maven-metadata.xml'

![image](https://user-images.githubusercontent.com/12250162/192998932-48dbd346-5a3d-4dc6-a5a8-1eb155600b21.png)

May I know is it any version updated? Many thanks!

**Comment by azahnen** [comment, marked as answer]
Windows was never officially supported as a runtime or development platform and due to recent changes ldproxy can no longer be run in a windows JRE/JDK at all. 

You have two options on windows:
- run ldproxy in a container with docker or similar
- if you want/need to run ldproxy directly in a JRE/JDK, you have to use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install)

**Reply by alankylee** [reply]
Thank you for your reply!
